terraform/github/teams.tf

```diff
@@ -250,6 +264,13 @@ locals {
       ]
     }
 
+    asdf-lab = {
```

Comment:
I think the name is a little too vague.

Reply:
Ah, ok, was just going with what it seemed every other team was doing. The plugin is called "lab" for the GitLab "lab" tool.

Reply:
OK